src/scss/components/switch/_index.scss

```diff
@@ -0,0 +1,185 @@
+@import "./config";
+
+hx-switch-control {
+  -ms-grid-column-align: start;
+  -ms-grid-columns: auto 1fr;
+  -ms-grid-rows: auto auto;
+  display: -ms-grid;
+  display: grid;
+  grid-template-areas:
+    "ctrl lbl"
+    ".    help";
+  grid-template-columns: auto 1fr;
+  grid-template-rows: auto auto;
+  justify-items: start;
+  grid-area: facade;
+  margin-top: 0.25rem; // FIXME: fix with proper inline styling
+  position: relative;
+  width: 68px;
+  height: 32px;
```

Comment:
Rather than use hardcoded pixels, we use `rem`'s or `em`'s.  Primarily we use `rem`.

```
1rem = 16px
``` 

Please update all pixel values.

Reply:
Changes have been done, please review.